src/views/Shipments.vue

```diff
@@ -104,7 +122,7 @@ export default defineComponent({
       const payload = {
         "inputFields": {
           "destinationFacilityId": this.user.facilityId,
-          "statusId": "PURCH_SHIP_SHIPPED",
+          "statusId": this.selectedSegment === "open" ? "PURCH_SHIP_SHIPPED" : "PURCH_SHIP_RECEIVED",
```

Comment:
Should this be available on all pages instead of just the shipments page? I think so right? @amansinghbais

Reply:
Yes, we can extend this approach to other pages as well to ensure consistency across the application. Initially, the implementation was focused on the shipment page as discussed, but we can apply the same for other pages as well.